app/classifier/tasks/combo/index.cjsx

```diff
@@ -65,20 +67,48 @@ ComboTask = React.createClass
       </g>
 
     PersistInsideSubject: (props) ->
+      currentComboAnnotations = []
+      allTaskTypes = props.classification.annotations.map (annotation) -> props.workflow.tasks[annotation.task].type
+      i = allTaskTypes.lastIndexOf('combo')
+      if i > -1
+        currentComboAnnotations = props.classification.annotations[i].value
       allComboAnnotations = []
+      allComboTypes = []
       props.classification.annotations.forEach (annotation) ->
         taskDescription = props.workflow.tasks[annotation.task]
         if taskDescription.type is 'combo'
           allComboAnnotations.push annotation.value...
+          annotation.value.forEach (a) ->
+            allComboTypes.push(props.workflow.tasks[a.task].type)
+
 
       <g className="combo-task-persist-inside-subject-container">
         {Object.keys(props.taskTypes).map (taskType) ->
           unless taskType is 'combo'
             TaskComponent = props.taskTypes[taskType]
             if TaskComponent.PersistInsideSubject?
+              # allComboAnnotations needs to be here so previous combo task annotations don't disappear
               fauxClassification =
                 annotations: allComboAnnotations
-              <TaskComponent.PersistInsideSubject key={taskType} {...props} classification={fauxClassification} />}
+                update: () => props.classification.update()
+              fauxChange = (annotation) ->
+                  props.onChange Object.assign({}, props.annotation, { value: currentComboAnnotations })
+              if props.annotation?.task? && props.workflow.tasks? && props.workflow.tasks[props.annotation.task]?.type is 'combo'
+                idx = allComboTypes.lastIndexOf(taskType)
+                if idx > -1
+                  # if the current annotation is for the combo task pass in the `inner` annotations
+                  fauxAnnotation = allComboAnnotations[idx]
+                else
+                  fauxAnnotation = props.annotation
+              else
+                fauxAnnotation = props.annotation
+              <TaskComponent.PersistInsideSubject
+                key={taskType}
+                {...props}
+                onChange={fauxChange}
```

Comment:
Marker selection is handled by `InsideSubject` and you can't drag unless a mark is selected. (as far as I can tell, I could be reading the code wrong...)